eth-contracts/.env.example

```diff
@@ -0,0 +1,20 @@
+
+# testnet-ropsten
+ROPSTEN_DEPLOYER_ACCOUNT="0x17c4b5CE0605F63732bfd175feCe7aC6b4620FD2"
+ROPSTEN_DEPLOYER_PRIVATE_KEY="0xc4ad657963930fbff2e9de3404b30a4e21432c89952ed430b56bf802945ed37a"
+ROPSTEN_API="https://ropsten.infura.io/v3/3ed3eadf912c4b31b800aafeedbf79eb"
+
+# testnet-kovan
+KOVAN_DEPLOYER_ACCOUNT="0x17c4b5CE0605F63732bfd175feCe7aC6b4620FD2"
+KOVAN_DEPLOYER_PRIVATE_KEY="0xc4ad657963930fbff2e9de3404b30a4e21432c89952ed430b56bf802945ed37a"
+KOVAN_API="https://kovan.infura.io/v3/3ed3eadf912c4b31b800aafeedbf79eb"
+
+# geth
+GETH_DEPLOYER_ACCOUNT="0x17c4b5CE0605F63732bfd175feCe7aC6b4620FD2"
+GETH_DEPLOYER_PRIVATE_KEY="0xc4ad657963930fbff2e9de3404b30a4e21432c89952ed430b56bf802945ed37a"
+
+
+# mainnet
+MAINNET_DEPLOYER_ACCOUNT="0x17c4b5CE0605F63732bfd175feCe7aC6b4620FD2"
```

Comment:
What is this for?
Seems like an insecure thing.

Reply:
I will remove this code